app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None | str:
+    if len(command.split()) != 3:
+        return "Incorrect statement printed"
+
+    cmd, curent_file, destination = command.split()
+
+    if cmd != "mv":
+        return "Command is incorrect"
+
+    dir_list = destination.split("/")
```

Comment:
Use os.path.split. Because in different systems there might be different path separator